city_scrapers/spiders/il_capital_development.py

```diff
@@ -0,0 +1,91 @@
+from datetime import datetime, timedelta
+
+from city_scrapers_core.constants import BOARD
+from city_scrapers_core.items import Meeting
+from city_scrapers_core.spiders import CityScrapersSpider
+
+
+class IlCapitalDevelopmentSpider(CityScrapersSpider):
+    name = "il_capital_development"
+    agency = "Illinois Capital Development Board"
+    timezone = "America/Chicago"
+    start_urls = ["https://www2.illinois.gov/cdb/about/boardmeetings"]
+    location = {
+        "name": "James R. Thompson Center",
+        "address": "100 West Randolph Street, 14th Floor, Chicago, IL 60601"
+    }
+
+    def parse(self, response):
+        """
+        """
+        for item in response.xpath("//tbody/tr"):
+            self._validate_location(item)
+            meeting = Meeting(
+                title="CDB Board Meeting",
+                description=self._parse_description(item),
+                classification=BOARD,
+                start=self._parse_start(item),
+                end=self._parse_end(item),
+                all_day=self._parse_all_day(item),
+                time_notes=self._parse_time_notes(item),
+                location=self.location,
+                links=self._parse_links(item),
+                source=self._parse_source(response),
+            )
+
+            meeting["status"] = self._get_status(meeting)
+            meeting["id"] = self._get_id(meeting)
+
+            yield meeting
+
+    def _parse_description(self, item):
+        """Parse or generate meeting description."""
+        description = item.xpath('.//td/text()').get()
+        for i in ["Chicago,", "Chicago", "Springfield", "&", "Collinsville"]:
+            if i in description:
+                description = description.replace(i, '')
+        description = description.lstrip()
+        return description
+
+    def _parse_start(self, item):
+        """Parse start datetime as a naive datetime object."""
+        start_date = item.xpath('.//th/text()').get()
+        start_time = item.xpath('.//th/text()[preceding-sibling::br]').get()
+        start = datetime.strptime(start_date + start_time, "%B %d, %Y %H:%M %p")
+        return start
+
+    def _parse_end(self, item):
+        """Parse end datetime as a naive datetime object. Added by pipeline if None"""
+        end_date = item.xpath('.//th/text()').get()
+        end_time = "12:00 AM"
+        end = datetime.strptime(end_date + end_time, "%B %d, %Y %H:%M %p")
+        end += timedelta(days=1)
+        return end
+
+    def _parse_time_notes(self, item):
+        """Parse any additional notes on the timing of the meeting"""
```

Comment:
The text currently getting pulled here makes more sense as input to `_get_status`. We can leave this as an empty string

Reply:
@pjsier - Should "time notes" and "description" be defined as empty strings directly in the Class attributes or as empty returns at the parse functions?

Reply:
It's probably easiest to just include them as empty strings in the `Meeting` kwargs. The parse methods are mostly there as a starter, so there's no reason to leave them in if they aren't useful

Reply:
@pjsier - when I set the Meeting kwargs to empty strings, and pass the xpath selector text to the `_get_status` `text` kwarg I get this test output:

```tests/test_chi_ssa_28.py:20: in <module>
    parsed_items = [item for item in spider.parse(test_response)]
tests/test_chi_ssa_28.py:20: in <listcomp>
    parsed_items = [item for item in spider.parse(test_response)]
city_scrapers/spiders/chi_ssa_28.py:29: in parse
    meeting["status"] = self._get_status(meeting)
../../.virtualenvs/city-scrapers-j7d0QWF1/lib/python3.7/site-packages/city_scrapers_core/spiders/spider.py:55: in _get_status
    if item["start"] < datetime.now():
E   TypeError: '<' not supported between instances of 'NoneType' and 'FakeDatetime'```

Reply:
ignore my deleted comment, my working branch wasn't clean with other changes.

Reply:
I'm not sure why that would have changed the `start` value if you only changed `time_notes` and `description`. Could you post a snippet of what was changed?